app/services/collators/disposable_income_collator.rb

```diff
@@ -23,12 +28,14 @@ def call
         total_outgoings_and_allowances: total_outgoings_and_allowances(attrs.monthly_cash_transactions_total),
         total_disposable_income: disposable_income(attrs.monthly_cash_transactions_total),
       )
+
+      Result.new(rent_or_mortgage_cash: attrs.rent_or_mortgage_cash)
     end
 
   private
 
     def outgoing_categories
-      CFEConstants::VALID_OUTGOING_CATEGORIES.map(&:to_sym) - [:child_care]
+      %i[maintenance_out legal_aid].freeze
```

Comment:
@starswan What do you think of setting up constant for outgoing categories? Eg: CFEConstants::VALID_DISPOSABLE_INCOME_OUTGOING_CATEGORIES